src/main/webapp/script.js

```diff
@@ -104,7 +104,7 @@ async function displayOverallResults(url) {
     displayWordCloudChart(wordFrequencyMap);
     hideLoadingGif();
     averageScore = commentStats.averageScore;
-    averageContainer.innerHTML = 'Average Sentiment Score: ' + averageScore;
+    averageContainer.innerHTML = 'Average Sentiment Score: ' + averageScore.toFixed(2);
```

Comment:
Can we also make this same change for the magnitude score in the tooltip strings?